application/views/css/form.css

```diff
@@ -129,19 +129,39 @@ form.renderable input[type="password"] {
 
 form.renderable-login input[type="text"],
 form.renderable-login input[type="number"],
-form.renderable-login input[type="password"] {
+form.renderable-login input[type="password"],
+form.renderable-login fieldset label[for="password-repeat"] {
 	height: 13px;
 	width: 235px;
 	padding: 6px 4px;
 }
 
 form.renderable-login fieldset label[for="username"],
 form.renderable-login fieldset label[for="password"],
-form.renderable-login fieldset label[for="auth_method"]{
+form.renderable-login fieldset label[for="auth_method"],
+form.renderable-login fieldset label[for="password-repeat"]{
 	display: inherit;
 	margin-left: 35px;
 }
 
+form.renderable-login .password-helper {
+	background-color: rgb(254, 244, 229);
+	height: 35px;
+	width: 275px;
+	border-bottom: 2px solid orange;
```

Comment:
Have you double checked with UX about this colour? They often have specific shades to use

Reply:
Yes, I have asked to Elin and she said it looks good.